src/api/config/initializers/airbrake.rb

```diff
@@ -102,9 +102,9 @@ def ignore_by_class?(notice)
   exceptions_to_ignore = ['ActiveRecord::RecordNotFound', 'ActionController::InvalidAuthenticityToken',
                           'CGI::Session::CookieStore::TamperedWithCookie', 'ActionController::UnknownAction',
                           'AbstractController::ActionNotFound', 'ActionView::MissingTemplate',
-                          'Timeout::Error', 'Net::HTTPBadResponse',
+                          'Timeout::Error', 'Net::HTTPBadResponse', 'Errno::ECONNRESET', 'Interrupt',
```

Comment:
`Errno::ECONNRESET` is when the remote host resets the connection request, so we cannot do anything about this.

`Interrupt` is when the interrupt signal is received, typically because someone pressed on Control-C inside a Rails console in the production server.